pkgs/development/python-modules/standard-distutils/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "standard-distutils";
```

Comment:
I would be more comfortable to use a more official source https://github.com/pypa/distutils, that is the source for the `distutils` that `setuptools` offer.